README.md

```diff
@@ -27,7 +27,7 @@ The resources directory contains Robot Keyword helpers for ``resincli`` and hard
 
 * Execute the following to run a example test against the qemux86-64 image you just downloaded.
```

Comment:
From line 18 to 28 - we can change resin-qemux86-64 to  resin-qemux86-64 or resin-qemux86 image.

and 22 can have both the examples.